redun/cli.py

```diff
@@ -1143,6 +1143,12 @@ def get_command_parser(self) -> argparse.ArgumentParser:
         # Run command.
         run_parser = subparsers.add_parser("run", allow_abbrev=False, help="Run a workflow task.")
         run_parser.add_argument("--no-cache", action="store_true", help="Do not use cache.")
+        run_parser.add_argument(
+            "--log-level",
+            default="INFO",
+            choices=log_levels.keys(),
+            help="Set redun logging level.",
+        )
```

Comment:
I think when I tried without this line I was getting an error. But that was quite some time ago and perhaps I just misused the CLI or something. Will try again without this code bit and come back with an update.

Reply:
OK If I remove this addition, then I get an error when I try to issue a run command with the `--log-level` argument. But I'm not sure why since it looks like the `run_parser` should inherit this argument from `subparsers`?

Reply:
Actually never mind. I've realized I had the `--log-level` argument after the `run` subcommand. Will remove this change.